spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,203 @@ class DeltaTable private[tables](
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   *
+   * Specifying properties here means that the target will override any properties with the same key
+   * in the source table with the user-defined properties.
+   *
+   * An example would be
+   * {{{
+   *  io.delta.tables.DeltaTable.clone(
+   *   "/some/path",
+   *   true,
+   *   Map("foo" -> "bar"))
+   * }}}
+   *
+   * @param target The path or table name to create the clone.
+   * @param replace Whether to replace the destination with the clone command.
+   * @param properties The table properties to override in the clone.
+   * @since 3.3.0
+   */
+  def clone(target: String, replace: Boolean, properties: Map[String, String]): DeltaTable = {
+    executeClone(table, target, replace, properties)
+  }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   * 
+   * An example would be
+   * {{{
+   *  io.delta.tables.DeltaTable.clone("/some/path", true)
+   * }}}
+   *
+   * @param target The path or table name to create the clone.
+   * @param replace Whether to replace the destination with the clone command.
+   * @since 3.3.0
+   */
+  def clone(target: String, replace: Boolean): DeltaTable = {
+    clone(target, replace, properties = Map.empty)
+  }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   * 
+   * An example would be
+   * {{{
+   *  io.delta.tables.DeltaTable.clone("/some/path")
+   * }}}
+   *
+   * @param target The path or table name to create the clone.
+   *
+   * @since 3.3.0
+   */
+  def clone(target: String): DeltaTable = {
+    clone(target, replace = false)
+  }
+
+  /**
+   * Clone a DeltaTable at a specific version to a given destination to mirror the existing
+   * table's data and metadata.
+   *
+   * Specifying properties here means that the target will override any properties with the same key
+   * in the source table with the user-defined properties.
+   *
+   * An example would be
+   * {{{
+   *  io.delta.tables.DeltaTable.cloneAtVersion(
+   *   5,
+   *   "/some/path",
+   *   true,
+   *   Map("foo" -> "bar"))
+   * }}}
+   *
+   * @param version The version of this table to clone from.
+   * @param target The path or table name to create the clone.
+   * @param replace Whether to replace the destination with the clone command.
+   * @param properties The table properties to override in the clone.
+   *
+   * @since 3.3.0
+   */
+  def cloneAtVersion(
+      version: Long,
+      target: String,
+      replace: Boolean,
+      properties: Map[String, String]): DeltaTable = {
+    executeClone(table, target, replace, properties, versionAsOf = Some(version))
+  }
+
+  /**
+   * Clone a DeltaTable at a specific version to a given destination to mirror the existing
+   * table's data and metadata.
```

Comment:
```suggestion
   * table's data and metadata.
   ```suggestion
   * table's data and metadata.
   *
   * Specifying properties here means that the target will override any properties with the same key
   * in the source table with the user-defined properties.
```
```